samples/pushBatchOfDocuments.ts

```diff
@@ -4,7 +4,10 @@ import {
   DocumentBuilder,
 } from '@coveo/push-api-client';
 async function main() {
-  const source = new PushSource('my_api_key', 'my_coveo_organization_id');
+  const source = new PushSource('my_api_key', 'my_coveo_organization_id', {
```

Comment:
Wasn't sure how far to go with the samples, so I decided to just add a few random-ish values to give the options some more visibility 😅